reporthandling/results/v1/reportsummary/frameworksummarymethods.go

```diff
@@ -51,7 +51,6 @@ func (frameworkSummary *FrameworkSummary) initResourcesSummary(controlInfoMap ma
 	for k, control := range frameworkSummary.Controls {
 		if statusInfo, ok := controlInfoMap[control.ControlID]; ok && statusInfo.InnerStatus != apis.StatusUnknown {
 			control.SetStatus(&statusInfo)
-			control.SetSubStatus(apis.SubStatusIntegration)
```

Comment:
question: why do we need to remove this in this PR?

Reply:
I can open a different PR for removing this, but this must be removed because there are other substatuses other than `SubStatusIntegration`.
The substatus is set in the `SetStatus` because it is nested in the `statusInfo` object.